File: backend/src/main/java/com/vencent/hotelmanagement/entity/Department.java
package com.vencent.hotelmanagement.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Id;
import javax.persistence.GeneratedValue;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name="department")
public class Department {

	@Id
	@GeneratedValue
	private int dept_id;
	
	private String dept_name;
	
	public int getDept_id() {
		return dept_id;
	}
	
	public void setDept_id(int dept_id) {
		this.dept_id = dept_id;
	}
	
	public String getDept_name() {
		return dept_name;
	}
	
	public void setDept_name(String dept_name) {
		this.dept_name = dept_name;
	}
	
}

File: frontend/src/app/shared/class/department.ts
export class Department {
    dept_id: string;
    dept_name: string;

    constructor(dept_id: string, dept_name: string){
        this.dept_id = dept_id;
        this.dept_name = dept_name;

    }

    set department_name(Name: string){
        this.dept_name = Name;
    }

    get department_name(): string{
        return this.dept_name;
    }

}
